include/alpaka/mem/view/Traits.hpp

```diff
@@ -353,4 +378,100 @@ namespace alpaka
     {
         return traits::CreateStaticDevMemView<TDev>::createStaticDevMemView(pMem, dev, extent);
     }
+
+    //! Creates a view to a pointer device
+    //!
+    //! \param pMem Pointer to memory.
+    //! \param dev Device from where pMem can be accessed.
+    //! \param extent Number of elements represented by the pMem.
+    //!               Using a multi dimensional extent will result in a multi dimension view to the memory represented
+    //!               by pMem.
+    //! \return A view to device memory.
+    template<typename TElem, typename TDev, typename TExtent>
+    auto createView(TElem* pMem, TDev const& dev, TExtent const& extent)
```

Comment:
This is subjective.
device being the second parameter is not logical imo. I would put it first, as the pointer and extent/pitch are interconnected

Reply:
I know I keep the order equal to `createStaticDevMemView`

Reply:
I will change the order but it will then have a different order than the direct creation of `ViewPlainPtrView`, never the less I prevare to have the device as the first param too.